cli/wallet/wallet.go

```diff
@@ -160,6 +164,20 @@ func NewCommands() []cli.Command {
 					},
 				},
 			},
+			{
+				Name:      "remove-account",
+				Usage:     "remove an account from the wallet",
+				UsageText: "remove-account --path <path> --addr <addr> [--force]",
+				Action:    removeAccount,
+				Flags: []cli.Flag{
+					walletPathFlag,
+					flags.AddressFlag{
+						Name:  "addr, a",
```

Comment:
And maybe make this one positional? `neo-go wallet remove Azlkjdfksjdflskwjsdf...` instead of `neo-go wallet remove -a Axlkfjsdfkjew...` seems a bit nicer to me.

Reply:
Done!